vllm/config.py

```diff
@@ -1741,6 +1741,9 @@ class is dynamically inherited by the worker class. This is used to inject
     rank: int = 0
     """Global rank in distributed setup."""
 
+    sleep_on_idle: bool = False
+    """Reduce CPU usage when sglang is idle."""
```

Comment:
```suggestion
    """Reduce CPU usage when vLLM is idle."""
```

Reply:
Fixed now, thanks. I should have reread more carefully.